tests/test_api.py

```diff
@@ -14,7 +14,14 @@
 
 @pytest.fixture(scope="class")
 def bliss_client(cfg, tmpdir_factory):
-    client = BlissClient(str(tmpdir_factory.mktemp("cwd")))
+    cwd = str(tmpdir_factory.mktemp("cwd"))
+    client = BlissClient(cwd)
+    mock_tests.MockSDSSDownloader(
+        run=cfg.datasets.sdss.run,
+        camcol=cfg.datasets.sdss.camcol,
+        fields=cfg.datasets.sdss.fields,
+        download_dir=cwd + "/data/sdss",
+    )
```

Comment:
@aakashdp6548 lmk if I should do it differently than call `MockSDSSDownloader` constructor in `bliss_client` like so, perhaps due to the overly implicit side effects of copying local files.

Reply:
I think it's fine, since it's in the fixture it should only be executed once